knime_extension/src/nodes/spatialnetwork.py

```diff
@@ -247,6 +270,7 @@ class GoogleDistanceMatrix:
         description="""Click [here](https://developers.google.com/maps/documentation/distance-matrix/get-api-key) for details on
         how to obtain and use a Google API key for the Distance Matrix API.""",
         default_value="",
+        # TODO: Check that API key is present
```

Comment:
See todo

Reply:
might be hard to check whether the key is valid